source/WebApp/scripts.ts

```diff
@@ -148,7 +170,8 @@ const build = task('build', async () => {
         icons(),
         manifest(),
         html(),
-        latest()
+        latest(),
+        asm()

```

Comment:
This one is a dependency for `ts` (before `tsLint`) instead of `build` (otherwise it might run too late).
Probably can be something like `tsAsmRegex` (similar to `tsLint`/`tsMain`).